internal/build/docker_builder.go

```diff
@@ -307,9 +307,9 @@ func (d *DockerBuilder) getDigestFromBuildOutput(ctx context.Context, reader io.
 }
 
 var dockerBuildCleanupRexes = []*regexp.Regexp{
-	// the "runc did not determinate sucessfully" just seems redundant on top of "executor failed running"
+	// the "runc did not determinate successfully" just seems redundant on top of "executor failed running"
 	// nolint
-	regexp.MustCompile("(executor failed running.*): runc did not terminate sucessfully"), // sucessfully (sic)
+	regexp.MustCompile("(executor failed running.*): runc did not terminate successfully"), // successfully (sic)
```

Comment:
i think this one needs to be reverted, because the typo is upstream (that's why it's marked `sic`)

Reply:
Oh, I see. Thank you for pointing it out.
I reverted. f800b65cfb954a5978b6961faa4305722ad1ba38